pages/Archivists/Format.md

```diff
@@ -13,18 +13,19 @@ menu: Archivists
 | 0.2     | Tentative |
 
 ## Purpose
-1. Specify the off-chain Archivist _storage_ format
-1. Enable development of an application that can permanently store, or cause to be permanently store necessary
-Liberty Protocol data in a uniform and systematic way.
+
+1. Specify the off-chain Archivist _storage_ format.
+1. Enable development of an application that can permanently store, or cause to be permanently store necessary Liberty Protocol data in a uniform and systematic way.
```

Comment:
This is awkward and not grammatically correct. Options:

1. Drop this sentence altogether. It boils down to "Enable archivists to do the thing archivists are supposed to do".
2. Drop the comma and store -> stored
3. Drop ", or cause to be permanently store" since causing to be stored is effectively storing.

Reply:
Dropped